scene/main/viewport.cpp

```diff
@@ -2944,13 +2948,12 @@ bool Viewport::_sub_windows_forward_input(const Ref<InputEvent> &p_event) {
 						gui.subwindow_drag = SUB_WINDOW_DRAG_CLOSE;
 						gui.subwindow_drag_close_inside = true; //starts inside
 						gui.subwindow_drag_close_rect = close_rect;
+					} else if (gui.subwindow_drag != SUB_WINDOW_DRAG_MOVE) {
+						gui.subwindow_drag = SUB_WINDOW_DRAG_START;
 					} else {
 						gui.subwindow_drag = SUB_WINDOW_DRAG_MOVE;
 					}
 
-					gui.subwindow_drag_from = mb->get_position();
-					gui.subwindow_drag_pos = sw.window->get_position();
```

Comment:
This line actually cause an error on master right now when dragging sub windows in single window mode. I think with the changes I made, these aren't necessary?
Stack trace:
```
[2] Window::get_position() const (/home/paul/Documents/git/godot/scene/main/window.cpp:82)
[3] Viewport::_sub_windows_forward_input(Ref<InputEvent> const&) (/home/paul/Documents/git/godot/scene/main/viewport.cpp:2952)
[4] Viewport::input(Ref<InputEvent> const&, bool) (/home/paul/Documents/git/godot/scene/main/viewport.cpp:3048 (discriminator 1))
[5] Window::_window_input(Ref<InputEvent> const&) (/home/paul/Documents/git/godot/scene/main/window.cpp:911 (discriminator 1))
[6] void call_with_variant_args_helper<Window, Ref<InputEvent> const&, 0ul>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/paul/Documents/git/godot/./core/variant/binder_common.h:197 (discriminator 6))
[7] void call_with_variant_args<Window, Ref<InputEvent> const&>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/paul/Documents/git/godot/./core/variant/binder_common.h:302)
[8] CallableCustomMethodPointer<Window, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/paul/Documents/git/godot/./core/object/callable_method_pointer.h:98)
[9] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/paul/Documents/git/godot/core/variant/callable.cpp:55)
[10] DisplayServerX11::_dispatch_input_event(Ref<InputEvent> const&) (/home/paul/Documents/git/godot/platform/linuxbsd/display_server_x11.cpp:2640)
[11] DisplayServerX11::_dispatch_input_events(Ref<InputEvent> const&) (/home/paul/Documents/git/godot/platform/linuxbsd/display_server_x11.cpp:2624)
[12] Input::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/paul/Documents/git/godot/core/input/input.cpp:615)
[13] Input::parse_input_event(Ref<InputEvent> const&) (/home/paul/Documents/git/godot/core/input/input.cpp:465)
[14] Input::flush_accumulated_events() (/home/paul/Documents/git/godot/core/input/input.cpp:858)
[15] DisplayServerX11::process_events() (/home/paul/Documents/git/godot/platform/linuxbsd/display_server_x11.cpp:2776)
[16] OS_LinuxBSD::run() (/home/paul/Documents/git/godot/platform/linuxbsd/os_linuxbsd.cpp:259)
[17] bin/godot.linuxbsd.tools.64(main+0x162) [0x1e016f8] (/home/paul/Documents/git/godot/platform/linuxbsd/godot_linuxbsd.cpp:60)
[18] /usr/lib/libc.so.6(__libc_start_main+0xd5) [0x7fa815bc1b25] (??:0)
[19] bin/godot.linuxbsd.tools.64(_start+0x2e) [0x1e014de] (??:?)
```